### ISSUE TITLE
Example data is missing

### ISSUE THREAD

**Issue by arokem** [tensorly/tensorly#497]
#### Describe the bug

Tensorly installed this morning using `pip install tensorly` into a fresh python 3.7 conda env on a Mac M1. 

The data in [this example](http://tensorly.org/stable/auto_examples/applications/plot_covid.html#sphx-glr-auto-examples-applications-plot-covid-py) does not seem to be where the code expects it to be.

#### Actual result

Raises: 

```
FileNotFoundError                         Traceback (most recent call last)
Cell In[1], line 8
      5 from matplotlib import pyplot as plt
      6 from matplotlib.cm import ScalarMappable
----> 8 data = load_covid19_serology()

File ~/miniconda3/envs/pca/lib/python3.8/site-packages/tensorly/datasets/data_imports.py:135, in load_covid19_serology()
    120 """
    121 Load an example dataset of COVID-19 systems serology.
    122 Formatted in a three-mode tensor of samples, antigens, and receptors.
   (...)
    131         Compromised humoral functional evolution tracks with SARS-CoV-2 mortality. Cell, 183(6), 1508-1519.
    132 """
    134 path_here = dirname(__file__)
--> 135 tensor = np.load(path_here + "/data/COVID19_data.npy")
    137 sampleLabels = np.array(
    138     [
    139         "Negative",
   (...)
    578     dtype=object,
    579 )
    580 antigenLabels = ["S", "RBD", "N", "S1", "S2", "S1 Trimer"]

File ~/miniconda3/envs/pca/lib/python3.8/site-packages/numpy/lib/npyio.py:405, in load(file, mmap_mode, allow_pickle, fix_imports, encoding, max_header_size)
    403     own_fid = False
    404 else:
--> 405     fid = stack.enter_context(open(os_fspath(file), "rb"))
    406     own_fid = True
    408 # Code to distinguish from NumPy binary files and pickles.

FileNotFoundError: [Errno 2] No such file or directory: '/Users/arokem/miniconda3/envs/pca/lib/python3.8/site-packages/tensorly/datasets/data/COVID19_data.npy'
```

**Comment by JeanKossaifi** [comment]
It seems the files aren't actually embedded in the release, this should be fixed by b648944 -- could you try installing directly from main and see if that resolves your issue? 

```bash
pip3 install git+https://github.com/tensorly/tensorly
```
If so I'll make a small release.

**Comment by arokem** [comment]
Yep! This now works with main. Thank you for the great software!
